### ISSUE TITLE
Add clarity when a article is read

### ISSUE THREAD

**Issue by An-anonymous-coder** [spacecowboy/Feeder#379]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/spacecowboy/feeder/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/spacecowboy/feeder/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

2.6.32

### Where did you get the app from

Other

### Feature or improvement you want

When an article has been read, the entire card's color should dim instead of the title being unbolded. 

### Why should this be added?

Currently it is very hard to see which articles have been read at a glance, since the difference between a bold and non-bold title is slim. This makes it much easier to tell, especially for users with accessibility needs. 

### Screenshots / Drawings / Technical details

Once a user reads an article, the entire card on the feeds page should be dimmed in color.

**Comment by spacecowboy** [comment]
I agree with the problem statement, but not necessarily the solution.

**Comment by shift18** [comment]
I'd like to see i high contrast theme, like this with a white card instead of blank when article is not read. (imagine dark text for the read article).
![feeder_contrast](https://github.com/user-attachments/assets/18292e9e-03f7-44d3-b539-61096ba44822)

**Comment by mpetuska** [comment]
Why not a simple checkbox indicator in the corner to indicate that it's read?

**Comment by shift18** [comment]
> Why not a simple checkbox indicator in the corner to indicate that it's read?

I'm not sure if that would look very clean, though personally  would prefer it compared to how it is now.

**Comment by Jiogo18** [comment]
Bold title is not enough when Bold Text is enabled in Android settings, because there is no difference when using Roboto (image below), and a difference too slight using System Default font.

With accessibility in mind (also considering a night mode, which makes color identification difficult),
an additive indicator is better, not just a color or font change.

I like the idea of strikethrough #613 because of the good "contrast" between read and unread.
However, I haven't found it recommended in M3, and it can be harder to read titles of read articles.
_(Yes, there is really no difference with bold Roboto on my phone.)_

<img width="360" height="559" alt="Image" src="https://github.com/user-attachments/assets/cdd480f2-0269-4cc1-afbb-283a55c1f74e" />

Ready to create a PR if you want to try it. It can come with an option too.

Any other indicator could be nice. If a checkbox doesn't look right, it could be a banner or a triangle in a corner? Either for read or unread.

To link with similar ideas
- #352 aims to change the opacity, which seems somewhat in line with what the disabled state of [Material Design Lists](https://m3.material.io/components/lists/specs#fecb8ca2-6836-40c1-b280-2d45995d4a24) does.
- #671 suggests using accent color. IMHO, it's the same idea as background color because of accessibility complications.

**Comment by spacecowboy** [comment]
Personally, I find it hard to picture something that works equally well for items with images and without.

**Comment by An-anonymous-coder** [comment]
Most other feed readers simply dim (technically add transparency) the text and images for read articles.

**Comment by spacecowboy** [comment]
The app already provides transparency to images in items which are read

**Comment by sconim** [comment]
@spacecowboy Many opinions, any real solutions? Is this a priority?

**Comment by spacecowboy** [comment]
> [@spacecowboy](https://github.com/spacecowboy) Many opinions, any real solutions? Is this a priority?

The app works great for me so it's not a priority for me.

But I do agree with that the contrast could be enhanced.

**Comment by sconim** [comment]
> > [@spacecowboy](https://github.com/spacecowboy) Many opinions, any real solutions? Is this a priority?
> 
> The app works great for me so it's not a priority for me.
> 
> But I do agree with that the contrast could be enhanced.

While, admittedly, the Compact Card view looks fine, in the sense that differences between the read & unread articles are both obvious and visually pleasing, the differences in all the other views are obscure, to say the least.

To make my point, I chose two similar views of Feeder (bottom) and another reader I use (top). The same article is marked as unread on both screenshots. Both readers are capable of showing text and pictures through different views but I purposely picked the simplest view. 
You tell me, which one would you rather be using. 

![Image](https://github.com/user-attachments/assets/194aa50e-07db-4b79-a40b-5322587e526e)
![Image](https://github.com/user-attachments/assets/342529cc-9279-4eb0-b968-25cb2d135715)

**Comment by spacecowboy** [comment]
I 100% agree the first is better. The question is only how to make it happen.

I'd consider adding special style here: https://github.com/spacecowboy/Feeder/blob/master/app/src/main/java/com/nononsenseapps/feeder/ui/compose/theme/Typography.kt

while also taking MaterialDesign into account: https://m3.material.io/styles/color/roles
